README.md

```diff
@@ -1 +1,8 @@
-# rg_instructor_analytics
\ No newline at end of file
+# rg_instructor_analytics
+
+##Unit tests
+All test runs in edx environment. 
```

Comment:
s/test runs in/tests could be run only in local/

Reply:
done